### PR TITLE
My adoptadog pages

### PR DESCRIPTION
**Pull request by ojspoko**
This is a pull request to merge my new pages into the repo

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <img src="images/adoptadog-logo.svg" alt="adoptadoglogo" width=150 height=150>
+      <img src="images/cart-icon.svg" alt="cart icon" width=50 height=50>
+      <h1>Seattle's premiere dog adoption agency</h1>
+    </header>
+    <nav><a href="index.html">Home </a> / <a href="dogs.html">Dogs</a> / <a href="blog.html">Blog</a></nav>
+  <section>
+    <h2>Adoptadog Blog</h2>
+    <div class="container">
+    <article>
+      <img class="img1" src="images/blog-1.jpg" alt="dog and girl grand canyon" width=300 height=300>
```
Comment (jimmynono) [inline blog.html]: take all inline CSS out of your HTML. We'll want to do it all in the CSS files.

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <header>
+      <img src="images/adoptadog-logo.svg" alt="adoptadoglogo" width=150 height=150>
+      <img src="images/cart-icon.svg" alt="cart icon" width=50 height=50>
+      <h1>Seattle's premiere dog adoption agency</h1>
+    </header>
+    <nav><a href="index.html">Home </a> / <a href="dogs.html">Dogs</a> / <a href="blog.html">Blog</a></nav>
+  <section>
+    <h2>Adoptadog Blog</h2>
+    <div class="container">
+    <article>
```
Comment (jimmynono) [inline blog.html]: This should be indented in one level from the div

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <div class="clearfix">
+      <h3>Traveling With Your Dog</h3>
+        <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.</p>
+        <p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p><br><br><br>
+      </div>
+    </article>
+    <article>
+      <img class="img1" src="images/blog-2.jpg" alt="dogs on leashes" width=300 height=300>
+      <div class="clearfix">
+      <h3>How To Walk Multiple Dogs</h3>
```
Comment (jimmynono) [inline blog.html]: Everything inside these divs should be indented in one level

File: checkout.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```
Comment (jimmynono) [inline checkout.html:4]: Don't forget that meta tag. You have it on at least one other page, and it'll need to be on all pages

File: checkout.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    <title>Adopt a Dog</title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" alt="adoptadoglogo" width=150 height=150>
+      <img src="images/cart-icon.svg" alt="cart icon" width=50 height=50>
+      <h1>Seattle's premiere dog adoption agency</h1>
+    </header>
+    <nav><a href="index.html">Home </a> / <a href="dogs.html">Dogs</a> / <a href="blog.html">Blog</a></nav>
```
Comment (jimmynono) [inline checkout.html]: The a tags should be on the next lines and indented in
```
<nav>
  <a href="#"></a>
  <a href="#"></a>
  <a href="#"></a>
</nav>
```

File: dogs.html [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    <header>
+      <img src="images/adoptadog-logo.svg" alt="adoptadoglogo" width=150 height=150>
+      <img src="images/cart-icon.svg" alt="cart icon" width=50 height=50>
+      <h1>Seattle's premiere dog adoption agency</h1>
+    </header>
+    <nav><a href="index.html">Home </a> / <a href="dogs.html">Dogs</a> / <a href="blog.html">Blog</a></nav>
+  <section>
+    <h2>Here are all of our Dogs!</h2>
+    <div class="grid-container">
+    <article class="profile">
```
Comment (jimmynono) [inline dogs.html]: All these articles should be indented in one level from the grid container

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+.header-item {
+  align-self:flex-end;
+}
+
+nav {
+  width: 100%;
+  background-color: rgb(217,217,217);
+  color:rgb(13,73,132);
+  padding:2%;
```
Comment (jimmynono) [inline styles.css]: Make sure you're consistent with formatting your css, you're missing spaces here and there.

```
h1 {
  color: black;
}
```

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.see-more {
+  clear:both;
+  text-align:center;
+}
+
+
+footer {
+  clear: both;
```
Comment (jimmynono) [inline styles.css]: Not sure these clears are needed since you're not floating anything.

File: javascript.js [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+ function Angus() {
+    alert("Angus, Bull Mastiff, $123.45")
+}
+
+ function Violet() {
+    alert("Violet, Chocolate Lab, $123.45")
+}
+
+ function Piper() {
```
Comment (jimmynono) [inline javascript.js]: Great! But all these functions are doing the same thing. A better way is to do it is pass the parametes in via your click handler and use the same function to do your work.  See an example at https://codepen.io/jimmynono/pen/RmbzQb?editors=1010

File: javascript.js [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+    alert("Luna, White Lab, $123.45")
+}
+
+  function Stella() {
+    alert("Stella, Chihuaha, $123.45")
+}
+
+let total = 0;
+
+  function adder(value) {
```
Comment (jimmynono) [inline javascript.js]: This whole function should be shifted to the left one indent level.

**Comment by MacaLaur** [comment]
[Peer Review html 200.docx](https://github.com/UWFront-End-Cert/adopt-a-dog-spring2019/files/3250889/Peer.Review.html.200.docx)

**Pull request closed without merging**